libr/util/pj.c

```diff
@@ -90,32 +90,42 @@ R_API PJ *pj_k(PJ *j, const char *k) {
 }
 
 R_API PJ *pj_kn(PJ *j, const char *k, ut64 n) {
-	pj_k (j, k);
-	pj_n (j, n);
+	if (n) {
+		pj_k (j, k);
+		pj_n (j, n);
+	}
 	return j;
 }
 
 R_API PJ *pj_kd(PJ *j, const char *k, double d) {
-	pj_k (j, k);
-	pj_d (j, d);
+	if (d) {
```

Comment:
same here: this check doesn't make sense to me. why D cannot be 0 ? it's not a pointer.